### ISSUE TITLE
GAN training curve share

### ISSUE THREAD

**Issue by quanhaol** [guandeh17/Self-Forcing#37]
Hi, great work!
Can you share your training curve when using R3GAN for training?
I have used R3GAN on my own network, and the pred real logits and fake logits all converge to 1 quickly, thus the R3GAN loss grad_norm quickly converges to 0.
Is this normal? How many steps do you use to converge when training R3GAN?
Thank you!
